dough-reflection/src/main/java/io/github/bakedlibs/dough/reflection/ReflectionUtils.java

```diff
@@ -41,8 +68,7 @@ private ReflectionUtils() {}
      *            The Method you are looking for
      * @return The found Method
      */
-    @Nullable
-    public static Method getMethod(@Nonnull Class<?> c, @Nonnull String method) {
+    public static @Nullable Method getMethod(@Nonnull Class<?> c, @Nonnull String method) {
         for (Method m : c.getMethods()) {
             if (m.getName().equals(method))
```

Comment:
I thought we had a thing against single-line methods without braces. Does this apply here?

Reply:
yup